v2/pkg/runner/options.go

```diff
@@ -159,6 +145,10 @@ func ParseOptions() *Options {
 		os.Exit(1)
 	}
 
+	if err := createProviderConfigYAML(defaultProviderConfigLocation); err != nil {
```

Comment:
We need to ensure that the file doesn't exist, otherwise the file with providers containing all the keys will be always overwritten at every run